test/test_simulation_execute.jl

```diff
@@ -15,11 +15,17 @@
     )
     build_out = build!(sim_single)
     @test build_out == PSI.BuildStatus.BUILT
-    execute_out = execute!(sim_single)
+    execute_out = execute!(sim_single, in_memory = in_memory)
     @test execute_out == PSI.RunStatus.SUCCESSFUL
 end
 
-@testset "2-Stage Decision Models with FeedForwards" begin
+@testset "Single stage sequential tests" begin
```

Comment:
This pattern doesn't add too much time and it is valuable.

Reply:
yeah, trimming down testing is important.